CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/ConstellationErrorManager.java

```diff
@@ -79,6 +86,71 @@ public void publish(final LogRecord errorRecord) {
+        if (ef instanceof InvocationTargetException ite) {
+            appendStackTrace(sbErrors, initialMessage, ite.getTargetException(), "    InvocationTargetException:\n  Caused By:\n");
+            appendCause(sbErrors, initialMessage, ite.getTargetException(), (depth + 1));
+        } else if (ef instanceof UndeclaredThrowableException ute) {
+            appendStackTrace(sbErrors, initialMessage, ute.getUndeclaredThrowable(), "    UndeclaredThrowableException:\n  Caused By:\n");
+            appendCause(sbErrors, initialMessage, ute.getUndeclaredThrowable(), (depth + 1));
+        } else if (ef instanceof ExceptionInInitializerError eiie) {
+            appendStackTrace(sbErrors, initialMessage, eiie.getException(), "    ExceptionInInitializerError:\n  Caused By:\n");
+            appendCause(sbErrors, initialMessage, eiie.getException(), (depth + 1));
+        } else if (ef instanceof PrivilegedActionException pae) {
+            appendStackTrace(sbErrors, initialMessage, pae.getException(), "    PrivilegedActionException:\n  Caused By:\n");
+            appendCause(sbErrors, initialMessage, pae.getException(), (depth + 1));
+        }
+    }
+    
+    private void appendStackTrace(final StringBuilder sbErrors, final String initialMessage, final Throwable ef, final String hierarchyMessage) {
+        final StringBuilder currentMessage = new StringBuilder("");
+        boolean addedDescription = false;
+        currentMessage.append(hierarchyMessage)
+                .append(ef.toString())
+                .append(SeparatorConstants.NEWLINE);
+        if (!sbErrors.toString().contains(currentMessage.toString()) && !initialMessage.trim().contains(currentMessage.toString().trim())) {
+            addedDescription = true;
+            sbErrors.append(currentMessage.append(SeparatorConstants.NEWLINE).toString());
+        }
+        
+        final StackTraceElement[] segmentElems = ef.getStackTrace();
+        if (segmentElems != null && segmentElems.length > 0) {
+            StringBuilder subStackTrace = new StringBuilder("");
+            for (int i = 0; i < segmentElems.length; i++) {
+                subStackTrace.append(segmentElems[i].toString())
+                        .append(SeparatorConstants.NEWLINE);
+            }
+            if (!sbErrors.toString().contains(subStackTrace.toString())) {
+                if (!addedDescription) {
+                    sbErrors.append(hierarchyMessage)
+                            .append(SeparatorConstants.NEWLINE);
+                }
+                sbErrors.append(subStackTrace.append(SeparatorConstants.NEWLINE).toString());
+            }
+        }
+    }
+    
+    @Override
+    public boolean isLoggable(final LogRecord logRec) {
+        final boolean firstCheck = super.isLoggable(logRec);
+        if (firstCheck && logRec.getThrown() != null) {
+            return false;
+        }
+        return firstCheck;
```

Comment:
this can all be replaced with `return super.isLoggable(logRec) && logRec.getThrown() == null`